homeassistant/components/point/alarm_control_panel.py

```diff
@@ -0,0 +1,84 @@
+"""
+Support for Minut Point.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/point/
+"""
+
+import logging
+
+from homeassistant.components.alarm_control_panel import (DOMAIN,
+                                                          AlarmControlPanel)
+from homeassistant.const import (STATE_ALARM_ARMED_AWAY, STATE_ALARM_DISARMED)
+from homeassistant.components.point.const import (
+    DOMAIN as POINT_DOMAIN, POINT_DISCOVERY_NEW)
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up a Point's alarm_control_panel based on a config entry."""
+    async def async_discover_home(home_id):
+        """Discover and add a discovered home."""
+        client = hass.data[POINT_DOMAIN][config_entry.entry_id]
+        async_add_entities([MinutPointAlarmControl(client, home_id)], True)
+
+    async_dispatcher_connect(
+        hass, POINT_DISCOVERY_NEW.format(DOMAIN, POINT_DOMAIN),
+        async_discover_home)
+
+
+class MinutPointAlarmControl(AlarmControlPanel):
+    """The platform class required by Home Assistant."""
+
+    def __init__(self, point_client, home_id):
+        """Initialize the entity."""
+        self._client = point_client
+        self._home_id = home_id
+
+    @property
+    def _home(self):
+        """Return the home object."""
+        return self._client.homes[self._home_id]
+
+    @property
+    def name(self):
+        """Return name of the device."""
+        return self._home['name']
+
+    @property
+    def state(self):
+        """Return state of the device."""
+        return STATE_ALARM_DISARMED if self._home[
+            'alarm_status'] == 'off' else STATE_ALARM_ARMED_AWAY
```

Comment:
We don't support triggered state?

Reply:
I can't find documentation for what the [alarm trigger](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/alarm_control_panel/__init__.py#L139) is supposed to do.

Reply:
I meant to ask if we know if the alarm is triggered or not. If we do, we should return that state here, when the alarm is triggered.

Reply:
The trigger service and method is meant to trigger the alarm on the device. But that is separate from knowing about the trigger state. Not all devices can do both and not all devices can do any of these.

Reply:
Ah, no. As I can see from the API-documentation is there no way to read triggered status (or get notifications of such).

 @archie, @themadinventor: Is there away to get a webhook when the alarm is triggered from the point-API?

Reply:
@MartinHjelmare can we add the current code now and update when the guys at Minut get back to me?

Reply:
Yes, of course.

Reply:
Fixed in https://github.com/home-assistant/home-assistant/pull/21207